invenio_circulation/api.py

```diff
@@ -70,9 +78,8 @@ def is_item_available(item_pid):
     if not cfg_item_available(item_pid):
         return False
 
-    if list(
-        Loan.get_loans(
-            item_pid,
+    if any(True for loan in Loan.get_loans_for_pid(
```

Comment:
This is probably something like
```
if any(Loan.get_loans_for_pid(...)):
```
As long as it returns an iterable, and I am not even sure it needs any()